moment-recur.js

```diff
@@ -590,6 +601,30 @@
             }
         };
 
+        // Checks for rule (by passing measure) and exception (by passing date)
+        Recur.prototype.has = function(dateOrRule) {
```

Comment:
I'm not a big fan of overloading this function to see if it has a rule or an exception.  Why not just call it ```hasRule``` and keep this really simple?